aggregator_api/src/lib.rs

```diff
@@ -405,10 +419,34 @@ mod models {
     use base64::{engine::general_purpose::URL_SAFE_NO_PAD, Engine};
     use janus_aggregator_core::task::{QueryType, Task};
     use janus_core::task::VdafInstance;
-    use janus_messages::{Duration, HpkeConfig, Role, TaskId, Time};
+    use janus_messages::{query_type::QueryTypeId, Duration, HpkeConfig, Role, TaskId, Time};
     use serde::{Deserialize, Serialize};
     use url::Url;
 
+    #[allow(dead_code)] // allowed in order to fully describe the interface and for later use
+    #[derive(Serialize, PartialEq, Eq, Debug)]
+    pub(crate) enum AggregatorRole {
+        Either,
+        Leader,
+        Helper,
+    }
+
+    #[derive(Serialize, PartialEq, Eq, Debug)]
+    pub(crate) struct AggregatorApiConfig {
+        pub dap_url: Url,
+        pub role: AggregatorRole,
+        pub vdafs: Vec<VdafId>,
+        pub query_types: Vec<QueryTypeId>,
+    }
+
+    #[derive(Debug, Clone, Copy, Eq)]
+    #[repr(u8)]
+    pub enum VdafId {
```

Comment:
```suggestion
    pub(crate) enum VdafId {
```
Consistent with the other models.